docker/remote-env/Dockerfile

```diff
@@ -62,6 +62,8 @@ ARG REMOTE_USER_SHELL=/bin/bash
 ARG REMOTE_USER_LOGIN_SCRIPT=$REMOTE_USER_HOME/.bashrc
 ARG REMOTE_USER_ID=3333
 ARG REMOTE_USER_GROUPS=sudo
+# ignoring because the current shell doesn't support pipefail; likely want to have a better fix long term
+# hadolint ignore=DL4006
 RUN useradd -lm \
```

Comment:
Maybe as long term fix exec it?
```sh
RUN ["/bin/bash", "-c", "set -o pipefail && useradd ..."]
```

Anyway, nit

Reply:
https://github.com/istio/ztunnel/issues/360

Reply:
I like the suggestion. I opened an issue to track on it. That exact syntax doesn't satisfy the linter but it may be worth using anyway to at least get the pipefail working.